hack/all-in-one/Dockerfile

```diff
@@ -10,7 +10,7 @@ USER root
 RUN if [ "$FROM_SOURCE" == "true" ]; then \
         mv microshift /usr/local/bin/microshift; \
     else \
-        export VERSION=$(curl -s https://api.github.com/repos/redhat-et/microshift/releases | grep tag_name | head -n 1 | cut -d '"' -f 4) && \
```

Comment:
Hey @zshi-redhat , thanks for the contribution! Instead of the `-L` switch, please instead change `redat-et` -> `openshift` in the curl links.

Reply:
@copejon Thanks for the review!
Updated the url to use `openshift`.

Reply:
Thank you Zenghui!